### ISSUE TITLE
[Bug] My beginner clues always seem to start with 1 step completed, until I manually check

### ISSUE THREAD

**Issue by Nick2bad4u** [SkyBouncer/clue_step_tracker#4]
This appears to only be happening with beginner clues.

https://gyazo.com/a6d6b87273b1a82299d1119c3b4618f3

**Comment by pelletier2017** [comment]
It seems like it happens when you open a beginner clue scroll bottle from fishing but not when picking them up off the ground.

**Comment by Nick2bad4u** [comment]
> It seems like it happens when you open a beginner clue scroll bottle from fishing but not when picking them up off the ground.

I only get beginners from birds nest, so maybe ur onto something.

**Comment by SkyBouncer** [comment]
Hmm, beginners and masters are wierd, but I'll look into it

**Comment by DivineInsanity** [comment]
Yep this is still an issue with Nest and bottle beginners atleast. Maybe a flag to set count to 0 if the player opens one of these?

**Comment by Xylot** [comment]
@SkyBouncer A possible solution would be subscribing to the `onMenuOptionClicked` event and ensuring a 0 count if the click event matched the beginner birdhouse item ID

[A similar example can be found here](https://github.com/runelite/runelite/blob/79ae5fdb2bdc6fe126c950d0f3f3041743e177b7/runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java#L1174)

**Comment by Nick2bad4u** [comment]
I'm cleaning up old issues I have open. Am I good to close this as not planned?
